Dockerfile

```
@@ -0,0 +1,13 @@
+FROM oven/bun:1.3.5
+
+WORKDIR /app
+
+COPY . .
+
+RUN HUSKY=0 bun install --frozen-lockfile
+RUN bun run build:wystack
```

Hadolint (2.15.1)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)